uint32 numberOfPerceivedObjects
PerceivedObject[] perceivedObjects
